sdk/eventhub/azure-eventhubs/tests/livetest/asynctests/test_receive_async.py

```diff
@@ -325,7 +325,9 @@ def batched():
 @pytest.mark.asyncio
 @pytest.mark.liveTest
 async def test_receive_run_time_metric_async(connstr_senders):
-    pytest.skip("Disabled for uamqp 1.2.2. Will enable after uamqp 1.2.3 is released.")
+    from uamqp import __version__ as uamqp_version
+    if uamqp_version < "1.2.3":
```

Comment:
not sure about whether the comparison between version string would work well in all case, but we could use the python built-in module:

from distutils.version import StrictVersion
if StrictVersion(uamqp_version) < StrictVersion('1.2.3'):

Reply:
Good point. Changed